server/src/routes/answerRoutes.js

```diff
@@ -5,12 +5,13 @@ import {
   deleteAnswer,
   getAnswer,
 } from "../controller/answerController.js";
+import { verifyToken } from "../middleware/auth.js";
 
 const answerRouter = Router();
 
-answerRouter.post("/", createAnswer);
-answerRouter.get("/:id", getAnswer);
-answerRouter.patch("/:id", updateAnswer);
-answerRouter.delete("/:id", deleteAnswer);
+answerRouter.post("/", verifyToken, createAnswer);
+answerRouter.get("/:id", verifyToken, getAnswer);
+answerRouter.patch("/:id", verifyToken, updateAnswer);
+answerRouter.delete("/:id", verifyToken, deleteAnswer);
```

Comment:
- should we import `verifyToken` to intex page, and just use it one time before routers inside `app.use()`instead of using it inside each router here or this gives better flexibility to handle each router individually?